inttest/backup/backup_test.go

```diff
@@ -155,6 +161,25 @@ func (s *BackupSuite) makeSnapshot(kc *kubernetes.Clientset) snapshot {
 	}
 }
 
+func (s *BackupSuite) VerifyFileSystemRestore() error {
+	ssh, err := s.SSH(s.ControllerNode(0))
+	if err != nil {
+		return err
+	}
+	defer ssh.Disconnect()
+
+	// Checking for containerd should be enough given https://github.com/k0sproject/k0s/issues/2420
+	// containerd may take a bit to start so we want to retry a few times
+	for i := 0; i < 18; i++ {
```

Comment:
You might also have a look at [`assert.Eventually`](https://pkg.go.dev/github.com/stretchr/testify/assert#Eventually).